### PR TITLE
Save & Commit commands

### PR DESCRIPTION
**Pull request by liorstauber**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command

Added firewall policy Save&Commit commands: 
  az network firewall policy draft create
  az network firewall policy draft update
  az network firewall policy draft intrusion-detection add
  az network firewall policy draft intrusion-detection remove
  az network firewall policy draft intrusion-detection list
  az network firewall policy rule-collection-group draft
  az network firewall policy rule-collection-group draft create
  az network firewall policy rule-collection-group draft update
  az network firewall policy rule-collection-group draft collection add-nat-collection
  az network firewall policy rule-collection-group draft collection add-filter-collection
  az network firewall policy rule-collection-group draft collection remove
  az network firewall policy rule-collection-group draft collection rule add
  az network firewall policy rule-collection-group draft collection rule update
  az network firewall policy rule-collection-group draft collection rule remove

### General Guidelines

- [X] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [X] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [X] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [X] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Thank you for your contribution! We will review the pull request and get back to you soon.

**Comment by yonzhan** [comment]
Please fix CI issues

**Comment by liorstauber** [comment]
> Please fix CI issues

fixed, can you help merging?

File: src/azure-firewall/setup.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 from codecs import open
 from setuptools import setup, find_packages
 
-VERSION = "1.0.1"
+VERSION = "1.1.0"
```
Comment (necusjz) [inline src/azure-firewall/setup.py:11]: mismatched in HISTORY.rst

**Comment by necusjz** [comment]
could you please provide your pr in the aaz repo?

**Pull request closed without merging**